src/PoolManager.sol

```diff
@@ -307,6 +311,17 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC6909Claims {
         emit ProtocolFeeUpdated(id, newProtocolFee);
     }
 
+    function collectProtocolFees(address recipient, uint256 id, uint256 amount)
```

Comment:
The more i think about it the more that makes sense... the fee controller doesnt care that theyre accruing as 6909s behind the scenes right, they just want to collect the current?

Reply:
Hmm good point. However I do think this relates back to the discussions we had a long time ago about 6909s in general, how we didn't want to write wrappers over all of the functions to enable calling them with `Currency` instead of `uint256` as per the standard. Right now in core we have a clear distinction where ERC20s use Currency (take and settle), and 6906s use the id (mint/burn). Since fees are explicitly accumulated as 6909s instead of ERC20s (they are claims to a balance left in core), I think it makes sense to have this be `id` instead of `currency`. Wdyt?

Reply:
Hmm right I remember this discussion and we left `mint` and `burn` to have `id`, which would then make sense to make this param an `id` unless we changed all of them to be Currency. I think I don't have a strong preferance either way.. I am forgetting why we felt strongly that mint and burn should be id?